packages/core/src/components/modal/test/default/modal.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from 'stencil-playwright';
```

Comment:
This test generates a second screenshot image once you branch out and run it on the local machine.
I guess it is due to some naming changes.
Please delete one that is not needed.